src/sentry/search/events/datasets/profile_functions.py

```diff
@@ -124,7 +124,7 @@ def _normalize(self, value: str) -> str:
 
         raise InvalidFunctionArgument(f"{value} is not a numeric column")
 
-    def get_type(self, value: str) -> str:
+    def get_type(self, value: str) -> str:  # type: ignore[override]  # baseclass is unsound
```

Comment:
there's at least a comment saying this is temporary in the source part.  the base class is taking `value: str | list[str]` due to the field aliasing thing that's happening there